### PR TITLE
Update roadmap docs

### LINKED ISSUE #1156: Update the Roadmap section of the website
For the planning process, we could direct to the [Contributing to the Project](https://submariner.io/development/contribution-guide/) page.

The _Future Work_ list items need to be refreshed

### PR DESCRIPTION
**Pull request by maayanf24**
Point to the more detailed planning process.
Remove future items list and point to the eh repo.

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

**Review by vthapar — APPROVED** [review]

File: src/content/community/roadmap/_index.en.md [hunk 1]
```diff
@@ -8,31 +8,21 @@ Submariner organizes all current and upcoming work using GitHub Issues, Projects
 
 ## Planning Process
 
-In preparation for sprint planning meetings (see [Submariner's Community Calendar][cal]), GitHub Issues should be raised for work that is to
-be a part of a sprint. Issues targeted for a sprint should be added to the upcoming Project's "Backlog" column. During sprint planning
-meetings, Issues will be discussed and moved to the "TODO" column and the milestone will be set to the targeted release. As contributors
-make progress during sprints, Issues should be moved through the "In Progress"/"Review/Verify"/"Done" columns of the Project. If an Issue is
-implemented during a release but additional work (like an ACK-fixed verification) tracked by the relevant Issue is necessary, the Issue can
-be carried over to the next Project but the Milestone should reflect where the code was shipped for accurate release note creation.
+For detailed explanation about Submariner Planning process, see [Submariner's Contributing to the project][contributing]
```
Comment (dfarrell07) [inline src/content/community/roadmap/_index.en.md]:
```suggestion
For a detailed explanation of Submariner planning process, see [Submariner's Contribution Guide][contributing].
```

File: src/content/community/roadmap/_index.en.md [hunk 2]
```diff
@@ -8,31 +8,21 @@ Submariner organizes all current and upcoming work using GitHub Issues, Projects
...
 ## Future Work
 
-Some high-level goals are summarized here, but the primary source for tracking future work are Submariner's GitHub Issues.
-
-* Network Policy across clusters (Coastguard)
-* Support for finer-grained connectivity policies (<https://github.com/submariner-io/enhancements/pull/4/files>)
-* Dynamic routing with BGP to support multi-path forwarding across gateways
-* Testing with multi-cluster Istio
-* Scale improvements
+Future work which is not planned for current release, can be found in the [enhancements] repository.
```
Comment (dfarrell07) [inline src/content/community/roadmap/_index.en.md]:
```suggestion
Future work which is not planned for the current release can be found in the [enhancements] repository.
```

I see the backlog board is closed. Are we not using it anymore? I've still been adding issues to it during scrub calls.

https://github.com/orgs/submariner-io/projects/15

Comment (maayanf24) [reply src/content/community/roadmap/_index.en.md]: @dfarrell07 - I have migrated the backlog board from the old project type to the new one, here is the new link: https://github.com/orgs/submariner-io/projects/29/views/1
See here: https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards

File: src/content/community/roadmap/_index.en.md [hunk 3]
```diff
@@ -8,31 +8,21 @@ Submariner organizes all current and upcoming work using GitHub Issues, Projects
...
 ## Future Work
 
-Some high-level goals are summarized here, but the primary source for tracking future work are Submariner's GitHub Issues.
-
-* Network Policy across clusters (Coastguard)
-* Support for finer-grained connectivity policies (<https://github.com/submariner-io/enhancements/pull/4/files>)
-* Dynamic routing with BGP to support multi-path forwarding across gateways
-* Testing with multi-cluster Istio
-* Scale improvements
+Future work which is not planned for the current release can be found in the [enhancements] repository.
```
Comment (dfarrell07) [inline src/content/community/roadmap/_index.en.md]:
```suggestion
Future work which is not planned for the current release can be found in the [backlog board](https://github.com/orgs/submariner-io/projects/29/views/1) and the [enhancements] repository.
```

Should we mention the backlog board here?

File: src/content/community/roadmap/_index.en.md (hunk 2 above)
Comment (dfarrell07) [reply src/content/community/roadmap/_index.en.md]: Ah okay, makes sense. Suggestion to link to it below.

**Comment by dfarrell07** [comment]
@maayanf24 This needs a manual rebase.

**Review by dfarrell07 — APPROVED** [review]

**Comment by maayanf24** [comment]
> @maayanf24 This needs a manual rebase.

Done

**Pull request merged**